<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Skynet Agent CLI Launcher v1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <!-- LEFT PANEL: Agent List -->
     <widget class="QFrame" name="leftPanel">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="minimumWidth">
       <number>250</number>
      </property>
      <property name="maximumWidth">
       <number>350</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="agentListLabel">
         <property name="text">
          <string>AVAILABLE AGENTS</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="agentListWidget">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="refreshButton">
         <property name="text">
          <string>üîÑ Refresh Agents</string>
         </property>
         <property name="minimumHeight">
          <number>35</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <!-- RIGHT PANEL: Agent Control and Output -->
     <widget class="QFrame" name="rightPanel">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <!-- Agent Info Section -->
       <item>
        <widget class="QGroupBox" name="agentInfoGroup">
         <property name="title">
          <string>Agent Information</string>
         </property>
         <property name="minimumHeight">
          <number>120</number>
         </property>
         <property name="maximumHeight">
          <number>150</number>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="agentNameLabel">
            <property name="text">
             <string>Name: -</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="agentPathLabel">
            <property name="text">
             <string>Path: -</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="agentTypeLabel">
            <property name="text">
             <string>Type: -</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="agentStatusLabel">
            <property name="text">
             <string>Status: STOPPED</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <!-- Control Buttons -->
       <item>
        <widget class="QFrame" name="controlFrame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="minimumHeight">
          <number>50</number>
         </property>
         <property name="maximumHeight">
          <number>60</number>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="startButton">
            <property name="text">
             <string>‚ñ∂Ô∏è START</string>
            </property>
            <property name="minimumHeight">
             <number>40</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stopButton">
            <property name="text">
             <string>‚èπÔ∏è STOP</string>
            </property>
            <property name="minimumHeight">
             <number>40</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="openLogsButton">
            <property name="text">
             <string>üìÇ Open Logs</string>
            </property>
            <property name="minimumHeight">
             <number>40</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearOutputButton">
            <property name="text">
             <string>üóëÔ∏è Clear</string>
            </property>
            <property name="minimumHeight">
             <number>40</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <!-- Output Terminal -->
       <item>
        <widget class="QGroupBox" name="outputGroup">
         <property name="title">
          <string>Agent Output (Live)</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPlainTextEdit" name="outputTextEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="lineWrapMode">
             <enum>QPlainTextEdit::NoWrap</enum>
            </property>
            <property name="maximumBlockCount">
             <number>10000</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionRefresh"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh Agents</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
